CODEOWNERS

```diff
@@ -2,4 +2,4 @@
 # the repo. Unless a later match takes precedence,
 # CODEOWNERS will be requested for
 # review when someone opens a pull request.
-*       @ulmen @ivanssimple @deveon @TokRa13 @pkatiushyn @wikarocha @johnmadden-sp @ksilp
+*       @simplepractice/infrastructure
```

Comment:
afair - this means after anybody from our team approve the PR - it will no more shown in https://github.com/pulls/review-requested for everybody else. Because github treats this as a "<username> approved on behalf of infra team".

Reply:
> That's only the case if two approvals

hm. If that's true that's good and I don't mind that also. Let's check.

> I think using the group is better because we won't have to update every repo when new people come or go.

Yeah that's understandable. Though in our team people come and go quite rare, luckily.
And tbh, it's not that hard to write some scripts like "add_infra.sh $1", "add_db.sh $1", etc. that will be adding or removing a person to all repos.